tox.ini

```diff
@@ -15,6 +15,13 @@ commands = flake8 {posargs}
 [testenv:venv]
 commands = {posargs}
 
+[testenv:typing]
+deps =
+    mypy
+    types-requests
+commands =
+    mypy --ignore-missing-imports {posargs} duecredit/
```

Comment:
I advice against using `--ignore-missing-imports`, as will also swallow errors about importing from a module that doesn't exist.  If the goal is to deal with dependencies that lack typing information, the better way is to add an override to the mypy config for each such dependency like so:

```ini
[mypy-packagename.*] 
ignore_missing_imports = True
```

Reply:
Would it make sense to ship mininimalistic .pyi type hints for cité citeproc and others ?
I mean as an interim solution ?

Reply:
I don't think so.